package.json

```diff
@@ -5,12 +5,13 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "babel src -d lib",
-    "coverage": "nyc --reporter=lcov npm run test",
+    "coverage": "jest --coverage",
     "dev": "babel -w src -d lib",
-    "lint": "eslint src test",
-    "fix": "eslint src test --fix",
-    "test": "mocha -t 20000",
-    "ci": "npm run lint && npm run test",
+    "lint": "eslint . --ignore-path .gitignore",
+    "fix": "npm run lint -- --fix",
+    "pretest": "npm run lint -- --cache",
```

Comment:
This is pretty big. Contributors that run tests locally shouldn't have to remember to _also_ `yarn run lint` to avoid silly lint errors breaking their PR builds.